lib/MixedModels/LMM.rb

```diff
@@ -165,6 +173,16 @@ def initialize(x:, y:, zt:, lambdat:, weights: nil, offset: 0.0, reml: true,
   #
   def LMM.from_formula(formula:, data:, weights: nil, offset: 0.0, reml: true, 
                        start_point: nil, epsilon: 1e-6, max_iterations: 1e6)
+    #Just out of curiousity, why don't you support the * and ||? Are you planning
+    #to implement them later? I'm not blaming you if you skipped them to keep
+    #the parser simpler, I'm just curious.
```

Comment:
Yes, I may implement those later (as well as / for nested random effects), but I think that the formula parser code would become too unclear if I do it now. I want to bring the existing code in a better form first (like separating it out in a different function for example, as you suggest).